linux_os/guide/services/ssh/ssh_server/disable_host_auth/rule.yml

```diff
@@ -54,7 +54,7 @@ references:
 template:
     name: sshd_lineinfile
     vars:
-        missing_parameter_pass: 'true'
+        missing_parameter_pass: 'false'
```

Comment:
This type of change has impact on tests as well. If we decide to merge them, we have reflect them on tests as well. For example, these tests would have to be rename with a suffix: `*.fail.sh`